.github/workflows/integration.yml

```
@@ -1,20 +1,76 @@
+#  push:
+#    branches:
+#      - main # main 브랜치에 머지될 때 트리거
+    workflow_dispatch: # 수동으로 트리거해서 테스트
 
 jobs:
-  integration:
+  springboot-integration: # Spring Boot 도커 이미지 생성 및 ECR 푸시
     runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)